src/storage/txn/scheduler.rs

```diff
@@ -340,6 +392,59 @@ fn process_read(cid: u64, mut cmd: Command, ch: SyncSendCh<Msg>, snapshot: Box<S
                 Err(e) => ProcessResult::Failed { err: e.into() },
             }
         }
+        Command::MvccByKey { ref ctx, ref key } => {
+            let mut reader = MvccReader::new(snapshot.as_ref(),
+                                             &mut statistics,
+                                             Some(ScanMode::Forward),
+                                             true,
+                                             None,
+                                             ctx.get_isolation_level());
+            let ts: u64 = u64::MAX;
+            match find_mvcc_infos_by_key(&mut reader, key, ts) {
+                Ok((lock, writes, values)) => {
+                    ProcessResult::MvccKey {
+                        mvcc: MvccInfo {
+                            lock: lock,
+                            writes: writes,
+                            values: values,
+                        },
+                    }
+                }
+                Err(e) => ProcessResult::Failed { err: e.into() },
+            }
+        }
+        Command::MvccByStartTs { ref ctx, start_ts } => {
+            let mut reader = MvccReader::new(snapshot.as_ref(),
+                                             &mut statistics,
+                                             Some(ScanMode::Forward),
+                                             true,
+                                             None,
+                                             ctx.get_isolation_level());
+            match reader.seek_first(start_ts)
+                .map_err(StorageError::from) {
+                Err(e) => ProcessResult::Failed { err: e.into() },
+                Ok(opt) => {
+                    match opt {
+                        Some(key) => {
+                            match find_mvcc_infos_by_key(&mut reader, &key, start_ts) {
```

Comment:
You should use commit_ts or `u64::MAX` as ts here,  otherwise it would return only those whose commit_ts is less than start_ts.